### ISSUE TITLE
on iOS sometimes the app will move into the dynamic island space 

### ISSUE THREAD

**Issue by brandonocasey** [advplyr/audiobookshelf-app#798]
### Steps to reproduce

1. Not entirely sure of the cause here, it happens when switching in and out of the app when reading ebooks, but not consistently. 
2.
3.

### Expected behaviour
The app should stay below the dynamic island/camera punchout on the screen

### Actual behaviour
The app moves up into the dynamic island space and the controls will be hard to use.

### Environment data

Audiobookshelf Version:

- [ ] Android App?
- [x] iOS App?

#### Android Issue
Android version:

Device model:

Stock or customized system:

#### iOS Issue
iOS Version: 0.9.65-beta

iPhone model: Iphone 14 pro max

**Comment by advplyr** [comment]
I believe this is related to reading an ebook. Are you using the ereader at all?

**Comment by brandonocasey** [comment]
Yeah this happens most of the time reading an ebook, but sporadically
